Tests/test_mode_bgra.py

```diff
@@ -0,0 +1,23 @@
+from helper import unittest, PillowTestCase
```

Comment:
I suppose, there is no need in the separate module. Please, move pack tests to `test_lib_pack.py: TestLibPack.test_pack` like unpack tests.